components/Wallet/PlaceOrder/Form/TradeInputs.jsx

```diff
@@ -196,15 +196,14 @@ export const TradeInputs = ({
         id="amount"
         name="amount"
         placeholder='0.00'
+        decimals={6}
```

Comment:
Hi @IYTEC , the asset needs to use the asset decimals, so wondering why this is now hardcoded to 6? For example an asset might have 0 decimals or 8 decimals.